public/assets-new-ui/styles.css

```diff
@@ -256,3 +261,225 @@ header[role="banner"] {
     }
   }
 }
+
+/*= TYPOGRAPHY */
+body {
+  color: #333;
+  font: normal 16px/1.5  'Guardian Text Egyptian Web', Georgia, serif;
+}
+
+h1 {
+  border-bottom: 1px solid #dcdcdc;
+  color: #121212;
+  font: bold 24px/1.25 'Guardian Egyptian Web', Georgia, serif;
+  margin: 0 -10px;
+  padding: 0 10px 12px;
+}
+ 
+input,
+select,
+textarea,
+button {
+  font: inherit;
+}
+  
+.blurb {
+  font-size: 14px;
+  line-height: 1.4;
+  margin: 3px 0 9px;
+}
+
+footer[role="contentinfo"] {
+  font-family: 'Guardian Text Sans Web', 'Helvetica Neue', Helvetica, Arial, 'Lucida Grande', sans-serif;
+}
+
+.license small {
+  font-size: 1em;
+}
+
+.support {
+  font-size: 14px;
+}
+
+form {
+  font-family: 'Guardian Text Sans Web', 'Helvetica Neue', Helvetica, Arial, 'Lucida Grande', sans-serif;
+}
+
+.form__radio-group--tabs {
+  font-family: 'Guardian Egyptian Web', Georgia, serif;
+  font-weight: bold;
+}
+
+/*= CONTRIBUTION FORM */
+.form__radio-group {
+  border: 0;
+  margin: 0;
+  padding: 0;
+}
+
+.form__radio-group legend {
+  display: none;
+}
+
+.form__radio-group ul {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.form__radio-group ul input {
```

Comment:
there are some CSS guidelines here: https://github.com/guardian/support-frontend/wiki/CSS

as a general rule, as per BEM methodology of avoiding having to think about specificity rules, we would try and keep things as flat as possible and just use class selectors

but this approach might be a temporary waypoint in your iterative process - if so, it's all good!

Reply:
sorry - me and @Ap0c literally got to this at the same millisecond

Reply:
Yeah it was, but now that I've sort of finished the big CSS work, I will add class names and move to BEM modifiers 👍